docs/plugins/google-cloud.md

```diff
@@ -134,22 +189,237 @@ This field specifies the timeout for the metrics export in milliseconds.
 
 #### disableMetrics
 
-Provides an override that disables metrics export while still exporting traces and logs.
+Provides an override that disables metrics export while still exporting traces
+and logs.
 
 #### disableTraces
 
-Provides an override that disables exporting traces while still exprting metrics and logs.
+Provides an override that disables exporting traces while still exprting metrics
+and logs.
 
 #### disableLoggingIO
 
 Provides an override that disables collecting input and output logs.
 
 #### forceDevExport
 
-This option will force Genkit to export telemetry and log data when running in the `dev` environment (e.g. locally).
+This option will force Genkit to export telemetry and log data when running in
+the `dev` environment (e.g. locally).
 
-> Note: When running locally, internal telemetry buffers may not fully flush prior to the process exiting, resulting in an incomplete telemetry export.
+> Note: When running locally, internal telemetry buffers may not fully flush
+prior to the process exiting, resulting in an incomplete telemetry export.
 
 ## Test your integration
 
-When configuring the plugin, use `forceDevExport: true` to enable telemetry export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace Explorer to view telemetry. Alternatively, navigate to the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) for an AI-idiomatic view of telemetry.
+When configuring the plugin, use `forceDevExport: true` to enable telemetry
+export for local runs. Navigate to the Google Cloud Logs, Metrics, or Trace
+Explorer to view telemetry. Alternatively, navigate to the
```

Comment:
Saying that Firebase is the alternative here feels odd to me given that this is for Firebase AI Monitoring.

Reply:
removed. we'll have the meta convo w/ the rest of the group.